addons/l10n_account_withholding/wizards/account_payment_register.py

```diff
@@ -0,0 +1,348 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+import json
+import logging
+from collections import defaultdict
+
+from odoo import Command, _, api, fields, models
+from odoo.exceptions import UserError, RedirectWarning
+from odoo.osv import expression
+
+_logger = logging.getLogger(__name__)
+
+
+class AccountPaymentRegister(models.TransientModel):
+    _inherit = 'account.payment.register'
+
+    # ------------------
+    # Fields declaration
+    # ------------------
+
+    l10n_account_wth_display_withholding = fields.Boolean(compute='_compute_l10n_account_wth_display_withholding')
+    l10n_account_wth_withhold_tax = fields.Boolean(
+        string='Withhold Tax Amounts',
+        compute='_compute_from_lines',
+        store=True,
+        readonly=False,
```

Comment:
Not readonly?

Reply:
No, the computation serves as a default here.
We enable it automatically if we have withholding taxes on the product, but if not, the user can still manually enable it.